capplets/network/mate-network-properties.c

```diff
@@ -392,61 +386,61 @@ setup_dialog (GtkBuilder *builder)
-			G_SETTINGS_BIND_DEFAULT);
+    g_settings_bind (ftp_proxy_settings, FTP_PROXY_PORT_KEY,
+                     gtk_builder_get_object (builder, "ftp_port_spinbutton"), "value",
+                     G_SETTINGS_BIND_DEFAULT);
+    g_settings_bind (ftp_proxy_settings, FTP_PROXY_HOST_KEY,
+                     gtk_builder_get_object (builder, "ftp_host_entry"), "text",
+                     G_SETTINGS_BIND_DEFAULT);
 
 	/* Socks */
-	g_settings_bind (socks_proxy_settings, SOCKS_PROXY_PORT_KEY,
-			gtk_builder_get_object (builder, "socks_port_spinbutton"), "value",
-			G_SETTINGS_BIND_DEFAULT);
-	g_settings_bind (socks_proxy_settings, SOCKS_PROXY_HOST_KEY,
-			gtk_builder_get_object (builder, "socks_host_entry"), "text",
-			G_SETTINGS_BIND_DEFAULT);
+    g_settings_bind (socks_proxy_settings, SOCKS_PROXY_PORT_KEY,
+                     gtk_builder_get_object (builder, "socks_port_spinbutton"), "value",
+                     G_SETTINGS_BIND_DEFAULT);
+    g_settings_bind (socks_proxy_settings, SOCKS_PROXY_HOST_KEY,
+                     gtk_builder_get_object (builder, "socks_host_entry"), "text",
+                     G_SETTINGS_BIND_DEFAULT);
 
 	/* Autoconfiguration */
-	g_settings_bind (proxy_settings, PROXY_AUTOCONFIG_URL_KEY,
-			gtk_builder_get_object (builder, "autoconfig_entry"), "text",
-			G_SETTINGS_BIND_DEFAULT);
+    g_settings_bind (proxy_settings, PROXY_AUTOCONFIG_URL_KEY,
+                     gtk_builder_get_object (builder, "autoconfig_entry"), "text",
+                     G_SETTINGS_BIND_DEFAULT);
 
-	g_signal_connect (gtk_builder_get_object (builder, "network_dialog"),
-			"response", G_CALLBACK (cb_dialog_response), NULL);
+    g_signal_connect (gtk_builder_get_object (builder, "network_dialog"),
+                      "response", G_CALLBACK (cb_dialog_response), NULL);
 
 	read_ignore_hosts_from_gsettings ();
 
 	model = create_listmodel();
 	populate_listmodel(GTK_LIST_STORE(model), ignore_hosts);
-	config_treeview(GTK_TREE_VIEW(gtk_builder_get_object (builder, "treeview_ignore_host")), model);
-
-	g_signal_connect (gtk_builder_get_object (builder, "button_add_url"),
-			  "clicked", G_CALLBACK (cb_add_url), builder);
-	g_signal_connect (gtk_builder_get_object (builder, "entry_url"),
-			  "activate", G_CALLBACK (cb_add_url), builder);
-	g_signal_connect (gtk_builder_get_object (builder, "button_remove_url"),
-			  "clicked", G_CALLBACK (cb_remove_url), builder);
+    config_treeview (GTK_TREE_VIEW (gtk_builder_get_object (builder, "treeview_ignore_host")), model);
+
+    g_signal_connect (gtk_builder_get_object (builder, "button_add_url"),
+                      "clicked", G_CALLBACK (cb_add_url), builder);
+    g_signal_connect (gtk_builder_get_object (builder, "entry_url"),
+                      "activate", G_CALLBACK (cb_add_url), builder);
+    g_signal_connect (gtk_builder_get_object (builder, "button_remove_url"),
+                      "clicked", G_CALLBACK (cb_remove_url), builder);
 }
```

Comment:
Do not fix the indent as it obfuscates the real change, and edit only the line : `GET_WIDGET ("network_dialog"));`.